_posts/en/2020-04-21-how-is-the-public-service-managing-large-IT-project.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title:  "How is the Public Service Managing Large IT Projects?"
+ref: how-is-the-public-service-managing-large-IT-project.md
+lang: en
+author: My Thuy Pham-Vo on behalf of the"IT Strategy Team"
+date: "2020-04-21"
+last_modified: "2020-04-21"
+excerpt_separator: <!--more-->
+---
+
+Having large projects raises concerns on the budget, available resources, the estimated success and the risks. Even if many kinds of research have informed the success of small projects are higher than large projects, many government departments and agencies still decide to proceed to implement large IT projects. As the Government of Canada works to improve its use of information technology during its transition towards digital, we have reviewed reports from the Office of the Audit General of Canada (OAG) on large IT projects undertaken and the lessons learned they provide.
```

Comment:
Maybe the introduction could include a follow-up of the previous blog post on Risks?
Something along the lines...

_In a previous [blog post](https://sara-sabr.github.io/ITStrategy/2019/12/20/why-we-are-promoting-risks.html), we highlighted that large changes and investments are increasing risks... Today, let's have a look at the history of large IT projects in the GC and how..._

Reply:
Ohhh that's a good one. It was hard to find a opening sentence.